packages/drafts-realm/chess.gts

```diff
@@ -0,0 +1,466 @@
+import {
+  Component as BoxelComponent,
+  CardDef,
+} from 'https://cardstack.com/base/card-api';
+import Component from '@glimmer/component';
+import StringCard from 'https://cardstack.com/base/string';
+import { contains, field } from 'https://cardstack.com/base/card-api';
+
+import lodash from 'https://cdn.jsdelivr.net/npm/lodash@4.17.21/+esm';
+import {
+  Chessboard,
+  FEN,
+} from 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/+esm';
+
+import Modifier from 'ember-modifier';
+
+type ChessboardModifierSignature = {
+  Args: {
+    Named: {
+      fen?: string;
+    };
+  };
+  Element: HTMLElement;
+};
+
+class ChessboardModifier extends Modifier<ChessboardModifierSignature> {
+  chessboard: any;
+  modify(
+    element: HTMLElement,
+    _positional: [],
+    { fen }: ChessboardModifierSignature['Args']['Named'],
+  ) {
+    debugger;
+    if (this.chessboard == undefined) {
+      this.chessboard = new Chessboard(element, {
+        position: fen ?? FEN.start,
+        style: {
+          pieces: {
+            file: 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/assets/pieces/standard.svg',
+          },
+        },
+      });
+    } else {
+      this.chessboard.setPosition(fen);
+    }
+  }
+}
+
+interface Signature {
+  Args: {
+    fen?: string;
+  };
+}
+
+class ChessboardComponent extends Component<Signature> {
+  <template>
+    <div id='board' {{ChessboardModifier fen=this.args.fen}}>
+    </div>
+    <style>
+      #board {
+        width: 600px;
+      }
+    </style>
+    {{! needs unscoped or global }}
+    <style unscoped>
```

Comment:
all this was copied unless you want it somewhere else. In addition, similar to our glimmer scoped css issue, we have to make use of seom unscopedness